nestkernel/connection_manager.cpp

```diff
@@ -1450,3 +1457,21 @@ nest::ConnectionManager::unset_have_connections_changed( const thread tid )
     have_connections_changed_[ tid ].set_false();
   }
 }
+
+void
+nest::ConnectionManager::set_has_get_connections_been_called( const thread tid )
+{
+  if ( has_get_connections_been_called_[ tid ].is_false() )
+  {
+    has_get_connections_been_called_[ tid ].set_true();
+  }
+}
+
+void
+nest::ConnectionManager::unset_has_get_connections_been_called( const thread tid )
```

Comment:
This function does not seem to be used?

Reply:
It's not, it's mostly there to make the interface consistent with that of `have_connections_changed_`. Maybe I should remove it?

Reply:
I also think that this unused function should best be removed.